packages/lodestar-cli/src/util/file.ts

```diff
@@ -16,10 +29,14 @@ export enum FileFormat {
   toml = "toml",
 }
 
+
+
 export function parse<T=Json>(contents: string, fileFormat: FileFormat): T {
   switch (fileFormat) {
     case FileFormat.json:
       return JSON.parse(contents);
+    case FileFormat.yaml:
+      return load(contents, {schema: yamlSchema});
```

Comment:
JSON is a subset of YAML, so actually you can load both with the same yaml.load function